apis/beacon/pool/attestations.v2.yaml

```diff
@@ -85,6 +85,9 @@ post:
             - type: array
               items:
                 $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Phase0.Attestation'
+      application/octet-stream:
+        schema:
+          description: "SSZ serialized `List[Attestation, MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]` or `List[SingleAttestation, MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]` bytes. Use content type header to indicate that SSZ data is contained in the request body."
```

Comment:
I wonder if it wouldn't be better to write something generic here that will be forward-compatible instead of listing exact object structures. Listing it this way adds maintenance because we will need to update this description whenever a new object is introduced in a fork. And this will have to be checked for all endpoints in each fork (assuming full SSZ support).

Reply:
The pint of a spec is to be specific, which is at least as much for the client as the server, i.e. to intentionally constrain the universe of valid responses and to ensure it's unambiguously testable. See also the other case you point out.

If anything, see elsewhere, just saying `Attestation` might be a bit on the vague, handwavy side.